pkg/controller/replication_controller.go

```diff
@@ -209,6 +211,14 @@ func (rm *ReplicationManager) getPodControllers(pod *api.Pod) *api.ReplicationCo
 		glog.V(4).Infof("No controllers found for pod %v, replication manager will avoid syncing", pod.Name)
 		return nil
 	}
+	if len(controllers) > 1 {
```

Comment:
> We need to be sorting these controllers in a consistent ordering (creation date, probably), and reporting that the controllers are conflicting

The sort on the reporting cannot be changed, because the first one is special in the rest of the code.